playbooks/bootstrap.yml

```diff
@@ -29,15 +29,22 @@
 #     debops bootstrap --sudo --limit host
 
 
-- name: Bootstrap host for Ansible management
+- name: Bootstrap Python support on a host
```

Comment:
The `debops.bootstrap/raw` only installs Python, so yes - Python support. The rest is in the next play.